client-js/app/elements/labrad-plot.ts

```diff
@@ -103,6 +104,9 @@ export class Plot extends polymer.Base {
 
   @property({type: String, value: 'zoomRect'})
   mouseMode: string;
+  
+  @property({type: Object, value: ''})
```

Comment:
Remove `value`. Also, isn't this actually set with an Array of axis descriptors, not an Object?